Add documentation for systemd per-service parsing configuration

seveas opened the pull request:
This documents the functionality added in https://github.com/fluent/fluent-bit/pull/9567

seveas committed:
Add documentation for systemd per-service parsing configuration

Signed-off-by: Dennis Kaarsemaker <dennis@kaarsemaker.net>

esmerel reviewed (approved):
minor edit suggestions

esmerel commented:
It also looks like the conflict can be resolved safely by accepting the incoming changes, but github has a bad habit of marking suggestions as outdated when you merge things so I'm not doing it myself this time.

eschabell committed:
Update pipeline/inputs/systemd.md

Co-authored-by: Lynette  Miles <6818907+esmerel@users.noreply.github.com>
Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell committed:
Update pipeline/inputs/systemd.md

Co-authored-by: Lynette  Miles <6818907+esmerel@users.noreply.github.com>
Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell committed:
Merge branch 'master' into systemd-parser

Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell committed:
docs: pipeline: inputs: systemd: fix Vale suggestions

  - Replace 'is not' with contraction 'isn't'
  - Reword 'unparsed message' to avoid Vale spelling suggestion

  Applies to #1508

Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell reviewed (approved):
@seveas just waiting on code PR merging.
